packages/server/graphql/mutations/helpers/notifications/SlackNotifier.ts

```diff
@@ -297,5 +297,121 @@ export const SlackNotifier: Notifier = {
 
   async integrationUpdated() {
     // Slack sends a system message on its own
+  },
+
+  async shareTopic(
+    dataLoader: DataLoaderWorker,
+    userId: string,
+    teamId: string,
+    meetingId: string,
+    reflectionGroupId: string,
+    stageIndex: number,
+    channelId: string
+  ) {
+    const r = await getRethink()
+    const [user, team, meeting, reflectionGroup, reflections, slackAuth] = await Promise.all([
+      dataLoader.get('users').load(userId),
+      dataLoader.get('teams').load(teamId),
+      dataLoader.get('newMeetings').load(meetingId),
+      dataLoader.get('retroReflectionGroups').load(reflectionGroupId),
+      r.table('RetroReflection').getAll(reflectionGroupId, {index: 'reflectionGroupId'}).run(),
+      r.table('SlackAuth').getAll(userId, {index: 'userId'}).filter({teamId})(0).run()
+    ])
+
+    if (!user) {
+      throw new Error('User not found')
+    }
+
+    if (!team) {
+      throw new Error('Team not found')
+    }
+
+    if (!slackAuth) {
+      throw new Error('Slack auth not found')
+    }
+
+    const {botAccessToken, slackUserId} = slackAuth
+    const manager = new SlackServerManager(botAccessToken!)
+    const channelInfo = await manager.getConversationInfo(channelId)
+
+    if (channelId !== slackUserId) {
```

Comment:
Why would the `channelId` be the same as the `slackUserId`? Isn't the `slackUserId` the `id` of the user?